app/components/ChangeUser.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { ScrollView, StyleSheet, TextStyle, View } from "react-native";
+import { LinkedAddress, UserDetail } from "../models/User";
+import { changeUser } from "../services/ApiService";
+import NotificationService from "../services/NotificationService";
+import SessionService from "../services/SessionService";
+import { Text, TouchableRipple } from "react-native-paper";
+import Colors from "../config/Colors";
+
+interface Props {
+  user?: UserDetail;
+  onSubmit: () => void;
+}
+
+const ChangeUser = ({ user, onSubmit }: Props) => {
+  const { t } = useTranslation();
+  const [linkedAddresses, setLinkedAddresses] = useState<LinkedAddress[]>([]);
+
+  useEffect(() => {
+    const availableAddresses = user?.linkedAddresses ?? [];
+    setLinkedAddresses(availableAddresses);
+  }, []);
+
+  const changeTo = (linkedAddress?: LinkedAddress) => {
```

Comment:
How can the linkedAddress be undefined?

Reply:
true was maybe an older implementation. is now not necessary anymore